web/src/pages/Route/components/Step1/RequestConfigView.tsx

```diff
@@ -376,6 +383,19 @@ const RequestConfigView: React.FC<RouteModule.Step1PassProps> = ({
           return null;
         }}
       </Form.Item>
+      <Form.Item
+        label={formatMessage({ id: 'page.route.service' })}
+        name='service_id'
+      >
+        <Select disabled={disabled}>
+          <Select.Option value=''>{}</Select.Option>
```

Comment:
how about using a global default value for the selector?  e.g `DEFAULT_SELECT_VALUE`

Then we could omit those fields using a global filter.

Reply:
should have an issue to track.